### ISSUE TITLE
Current time on level

### ISSUE THREAD

**Issue by freddogg23** [fluid-queue/fluid-queue#121]
It would be nice to have a current time on the level whenever you trigger the !current command, and maybe even a total time after the current time for as long as you have a level queued

**Comment by demize** [comment]
This is blocked by #122 (which will allow us to keep track of level history). Shouldn't be too much work to add in once that's done, so I'd expect this to make it into the 2.0.0 release.

**Comment by liquidnya** [comment]
This can be done within the current `develop` branch as well.
It could be implemented the same way weight is calculated per used, which happens every minute.
The benefits of this approach:

- Weight is only calculated while the stream is online and the current time of a level should also only increase while the stream is online
- Changing the weight is causing the save file to be written and therefore saving the queue after changing the current time of a level doesn't cost more since writing the new save file already happens every minute - so increasing the current time can happen while weights are being updated, but before the save file is written to disk

The save file already contains a `current` entry which could be easily extended with this:
```ts
current: SubmittedEntryV3_1.extend({
  playTime: z.object({ minutes: z.number().int().nonnegative() }),
})
  .nullable()
  .describe("the currently selected queue entry or null"),
```

I think I could work on this on `develop` since it was requested in my stream recently.

**Comment by liquidnya** [comment]
Displaying the current time since the level was picked is possible in https://github.com/fluid-queue/fluid-queue/tree/feature/current-time, but displaying the total time a level took is not possible with the current changes and that part is blocked by #122.

I will create a PR after adjusting the tests and even if the PR is merged we can keep this ticket open until the total time is implemented.
